numpy-mypy/numpy/__init__.pyi

```diff
@@ -0,0 +1,231 @@
+from typing import Any, Dict, Iterator, List, Optional, Sequence, Tuple, Union
+
+class dtype: ...
+_dtype_alias = dtype
+
+
+class flagsobj(Dict[str, bool]):
+    aligned = None       # type: bool
+    behaved = None       # type: bool
+    c_contiguous = None  # type: bool
+    carray = None        # type: bool
+    contiguous = None    # type: bool
+    f_contiguous = None  # type: bool
+    farray = None        # type: bool
+    fnc = None           # type: bool
+    forc = None          # type: bool
+    fortran = None       # type: bool
+    owndata = None       # type: bool
+    updateifcopy = None  # type: bool
+    writeable = None     # type: bool
+
+
+# TODO Complete scalar hierarchy
+class generic: ...
+class bool_(generic): ...
+class object_(generic): ...
+class number(generic): ...
+class flexible(generic): ...
+
+class ndarray:
```

Comment:
It may be useful to have 2 different generic arguments for the type:
* the index type (which can be int, Tuple[int, int], etc)
* the element type (which should be the python type mapped to the elements)